cypress/support/commands.ts

```diff
@@ -37,29 +37,36 @@
 // }
 
 // https://github.com/nextauthjs/next-auth/discussions/2053#discussioncomment-1191016 :pray::pray::pray:
-Cypress.Commands.add("login", () => {
-	cy.intercept("/api/auth/session", { fixture: "session.json" }).as("session");
-	console.log(Cypress.env());
-	// Set the cookie for cypress.
-	// It has to be a valid cookie so next-auth can decrypt it and confirm its validity.
-	// This step can probably/hopefully be improved.
-	// We are currently unsure about this part.
-	// We need to refresh this cookie once in a while.
-	// We are unsure if this is true and if true, when it needs to be refreshed.
-	cy.setCookie("next-auth.session-token", Cypress.env("SESSION_TOKEN"));
-	cy.setCookie(
-		"__Secure-next-auth.session-token",
-		Cypress.env("SESSION_TOKEN"),
-		{ secure: true }
-	);
+Cypress.Commands.add('login', () => {
+  cy.intercept('/api/auth/session', { fixture: 'session.json' }).as('session');
+  console.log(Cypress.env());
+  // Set the cookie for cypress.
+  // It has to be a valid cookie so next-auth can decrypt it and confirm its validity.
+  // This step can probably/hopefully be improved.
+  // We are currently unsure about this part.
+  // We need to refresh this cookie once in a while.
+  // We are unsure if this is true and if true, when it needs to be refreshed.
+  cy.setCookie('next-auth.session-token', Cypress.env('SESSION_TOKEN'));
+  cy.setCookie(
+    '__Secure-next-auth.session-token',
+    Cypress.env('SESSION_TOKEN'),
+    { secure: true }
+  );
+});
+Cypress.Commands.add('mock', () => {
```

Comment:
```suggestion
Cypress.Commands.add('mockAPI', () => {
```

can you rename this to mockAPI and make according changes in the test functions

Reply:
oops bad variable naming , will fix :+1: